### PR TITLE
nightelf magical eyes

### PR DESCRIPTION
**Pull request by icelille999**
## Changelog:
- Added golden eyes for Night Elf race that have natural magic traits (Otherworldly, Sorcerous, Magical, Magi) 

## Developer changelog:
- Tied magic_good traits to night elf trait in 00_trait_modifiers.txt

# How to test:
If using the Ruler editor, assign magic traits (Otherworldly, Sorcerous, Magical, Magi) to character. Eyes will not immediately change until you accept your character- the night elf trait doesn't get assigned to a character until outside of ruler editor. 
To make a night elf, make sure character's culture is in the sub group "Darnassian", which is under the Elven culture. This also forces a character to gain the night elf trait (despite their appearance or apparent ethnicity.)
Otherwise, any night elf in game already should already inherit these eyes. Later gameplay needs testing to confirm offspring heritability, how culture mixing works out, etc.

### CODE REVIEW

File: gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt [hunk 1]
```diff
@@ -1204,3 +1204,28 @@ drunkard = {
...
+			magic_good_2
+			magic_good_3
+			magic_good_noninheritable
+			}
+		dna_modifiers = {
+			color = {
+			gene = eye_color
+			mode = replace
+			x = 0.11
+			y = 0.3
```
Comment (zumbak04) [inline gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt:1226]: The current color feels too orange to me.
```suggestion
			x = 0.19
			y = 0.5
```

File: gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt [hunk 2]
```diff
@@ -1204,3 +1204,28 @@ drunkard = {
 		}
 	}
 }
+
+magic_good = {
```
Comment (zumbak04) [inline gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt:1208]: You should visit `wc_ethnicities_elven.txt` and remove the random gold eye color option from the night elves.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt (hunk 2 above)
Comment (icelille999) [reply gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt:1209]: I knew you mentioned this was in the original code, but I couldn't find it when I was looking around. Thank you for pointing that out! I will update this in my next commit.

File: gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt (hunk 1 above)
Comment (icelille999) [reply gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt]: Couldn't add in batch commit, so I will add this manually into next commit. Looking at other pictures of in game WoW characters, I agree. I thought maybe they were more amber, but they are really much more yellow.

File: gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt (hunk 2 above)
Comment (icelille999) [reply gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt:1209]: Shouldn't this also be deleted from the highborne?

**Review by zumbak04 — APPROVED** [review]
LGTM!

File: gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt [hunk 3]
```diff
@@ -1204,3 +1204,31 @@ drunkard = {
...
+			OR = {
+				has_trait = creature_night_elf
+				has_trait = creature_highborne
+			}
+		}
+		traits = {
+			magic_good_1
+			magic_good_2
+			magic_good_3
+			magic_good_noninheritable
```
[the remark this answers is not in the document]
Comment (MemeWizard) [reply gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt:1223]: Nah, it's the other trait

**Pull request merged**